docs/version-1.4.0.md

```diff
@@ -0,0 +1,78 @@
+# Version 1.4.0
+
+**Release Name**: CTK 1.4.0
+
+**Upgrade From**: CTK 1.3.0
+
+**Support**: Stable Release
+
+**Release Date**: 15th April, 2024
+
+## Overview
+
+The 1.4.0 version of CTK includes the following new features:
+
+**New Features in CTK 1.4.0 release** 
+1. Quality Assessment Report is now available for Quality Assessment Collections.
+   This report can be submitted for review just like Compliance Collection report. 
+2. Added a new CTK Landing page
+3. Added Terms & Conditions consent popup for partners on login. If the consent is not given then the partner will be logged out.
+
+**SBI Testcases Enhancements**
+1. Added a negative SBI testcase SBI1196 where Discover request attributes are in UPPER CASE
+2. Added testcases(SBI1197, SBI1198 and SBI1199) where bioSubTypes is optional in RCapture request
+3. Enhanced all SBI schemas by adding "Additional Properties" as "false" in all nested objects as well. This will disallow any extra attributes at nested levels.
+4. Community reported issue in the CTK UI GitHub repository is fixed - Android SBI CTK Check Device Status failed: SBI1028, SBI1029
+5. In trust validation also included a check for Org Name for logged in user and the Subject Org in the certificate.
+6. SBI RCapture response will be encrypted saved in CTK DB
+7. As per community reported issue, CTK is now sending "previousHash" as SHA256 hash of an empty UTF-8 string instead of just an empty string.
```

Comment:
In response to a community-reported issue, CTK now sends the ‘previousHash’ as the SHA256 hash of an empty UTF-8 string, rather than simply an empty string.